Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.h

```diff
@@ -163,7 +163,8 @@ class WebsiteDataStore : public API::ObjectImpl<API::Object::Type::WebsiteDataSt
 #if PLATFORM(IOS_FAMILY)
     String cookieStorageDirectory();
     String containerCachesDirectory();
-    static String containerTemporaryDirectory();
+    String containerTemporaryDirectory();
```

Comment:
Seems like this should be const?

Probably applies to some/most of the other getters too.

Reply:
Updated the patch with const.